src/DynamoCoreWpf/Views/Core/GeometryScalingPopup.xaml.cs

```diff
@@ -0,0 +1,113 @@
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using Dynamo.ViewModels;
+using ViewModels.Core;
+using System.Linq;
+using Dynamo.Logging;
+using System;
+using Res = Dynamo.Wpf.Properties.Resources;
+using System.Windows.Input;
+
+namespace Dynamo.Views
+{
+    public partial class GeometryScalingPopup : Popup
+    {
+        private GeometryScalingViewModel viewModel;
+        private DynamoViewModel dynamoViewModel;
+        
+        public GeometryScalingPopup(DynamoViewModel dynViewModel)
+        {
+            InitializeComponent();
+
+            var defaultWorkspace = dynViewModel.Workspaces.FirstOrDefault();
+
+            if (defaultWorkspace != null && viewModel == null && defaultWorkspace.GeoScalingViewModel != null)
+                viewModel = defaultWorkspace.GeoScalingViewModel;
+            DataContext = viewModel;
+            dynamoViewModel = dynViewModel;
+        }
+
+        /// <summary>
+        /// This event is generated every time the user clicks a Radio Button in the Geometry Scaling section
+        /// The method just get the Radio Button clicked and saves the ScaleValue selected
+        /// This are the values used for the scales:
+        /// - 2 - Small
+        ///   0 - Medium (Default)
+        ///   2 - Large
+        ///   4 - Extra Large
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void Geometry_Scaling_Checked(object sender, System.Windows.RoutedEventArgs e)
+        {
+            var selectedButton = sender as Button;
+            if (selectedButton == null) return;
+            //var buttons = BaseGrid.Children.OfType<Button>();
```

Comment:
if not needed, let's delete it

Reply:
deleted in the next commit: https://github.com/DynamoDS/Dynamo/pull/13656/commits/c20979418fa67d8906f184af3810952b71d26641